portfolio/src/main/webapp/favorite_things.html

```diff
@@ -8,191 +8,162 @@
     <script src="script.js"></script>
 
     <style>
-        /*style the header*/
-        header
-        {
-            background-color:Plum;
-            padding-top: 15px;
-            padding-bottom:5px;
+        /*style the h2 header*/
+        h2{
             text-align:center;
             font-family: 'Berkshire Swash', cursive;
-            font-size: 40px;
+            /*padding-top:30px;*/
+            font-size: 80px;
             color:White;
-            border-bottom-style: solid;
-            border-bottom-color: black;
         }
         /*style the  body*/
-        body
-        {
-            background-color:white;
-        }
-        img.rounded-corners
-        {
+        body{
+            background-color:White;
+            margin:0;
+            }
```

Comment:
minor comment, I noticed that the end braces are inconsistent: sometimes they are at the same indentation as the open brace, sometimes they are more indented. I found a handy css + html style guide here: go/htmlcssstyle. The example code shows the end brace indentation matching the start brace. Move this one back a bit. Same applies to the rest of the braces